model/schema/allele_dto.yaml

```diff
@@ -0,0 +1,540 @@
+      collection vocabulary' Vocabulary
+    domain: AlleleDTO
+    range: string # CV 'Allele collection vocabulary'
+
+  inheritance_mode_name:
+    description: >-
+      Name of VocabularyTerm describing the inheritance mode from the 'Allele
+      inheritance mode vocabulary', e.g. dominant / semi-dominant /
+      recessive / unknown / codominant
+    domain: AlleleInheritanceModeSlotAnnotationDTO
+    range: string # CV 'Allele inheritance mode vocabulary'
+
+  integration_method_name:
+    description: >-
+      WormBase captures the method by which an extrachromosomal transgene was integrated into the genome.
+    domain: GenerationMethodDTO
+    range: string  #CV 'WB Transgene Integration Methods'
+    multivalued: false
+
+  irradiation_mutagen_name:
+    description: >-
+      The name of the irradiation used to generate the mutation through mutagenesis
+    domain: GenerationMethodDTO
+    range: string # CV 'Irradiation Mutagens'
+
+  laboratory_of_origin_curie:
+    description: >-
+      The curie of the laboratory of origin for the entity.
+    domain: AlleleDTO
+    range: string
+
+  molecular_mutation_names:
+    description: >-
+      Name of the VocabularyTerm describing the molecular mutation
+    domain: AlleleMolecularMutationSlotAnnotationDTO
+    range: string
+    multivalued: true
+    required: true
+
+  mutagenesis_method_names:
+    description: >-
+      Name of the VocabularyTerm describing the mutagenesis method, e.g.
+      spontaneous / naturally occurring / radiation-induced / recombinant /
+      ENU / CRISPR / TALEN / gamma rays / not specified / spontaneous / DNA /
+      DNA AND CRISPR / DNA and TALEN / zinc finger nuclease / EMS
+    domain: AlleleGenerationMethodAssociationDTO
+    range: string
+    multivalued: true
+
+  mutation_target_strain_curie:
+    description: >-
+      Curie of the particular strain that is targeted by the generation method
+      for a particular allele (MGI only)
+    domain: AlleleGenerationMethodAssociationDTO
+    range: string
+
+  mutation_type_curies:
+    description: Curies of SOTerms representing mutation type
+    domain: AlleleMutationTypeSlotAnnotationDTO
+    range: string
+    multivalued: true
+
+  nomenclature_event_name:
+    description: >-
+      Name of the VocabularyTerm describing the nomenclature event
+    domain: AlleleNomenclatureEventSlotAnnotationDTO
+    range: string
+    required: true
+
+  object_allele_curie:
+    description: >-
+      The curie of the allele that is acting as the object of an AlleleAlleleAssociation
+    domain: AlleleAlleleAssociationDTO
+    range: string
+
+  transgene_chromosome_location_curie:
+    description: >-
+      The curie string of the chromosome to which a transgene has been mapped. Used for WormBase
+      transgenes that have been integrated into the genome and mapped to a chromosome.
+    domain: AlleleDTO
+    range: string
+    multivalued: false
+
+## ------------
+## ALLELE DTO SLOT ANNOTATION SLOTS
```

Comment:
I’m not sure we should separate these from other slots.  It’s not immediately obvious from the slot name that these slots have a range of a SlotAnnotation.  Also, these are slots of the Allele class not the SlotAnnotation classes - my interpretation of ‘Slot Annotation Slots’ would be slots used in the SlotAnnotation classes themselves (eg. mutation_type_curies).  I think just listing all slots alphabetically would be more intuitive.

Reply:
@markquintontulloch 
You're right, sorry, I thought it made a lot more sense earlier when I was trying to compare the DTO and non-DTO yaml files side-by-side. I've fixed in my latest commit.

Reply:
Actually, not all done yet...

Reply:
OK, now fixed